docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -776,6 +778,46 @@ In this example, the table is partitioned by the month of ``order_date``, a hash
         country VARCHAR)
     WITH (partitioning = ARRAY['month(order_date)', 'bucket(account_number, 10)', 'country'])
 
+Sorted tables
+-------------
+
+The connector supports sorted files as a performance improvement. Data is sorted during writes
+within each file based on the specified array of one or more columns.
+
+Sorting is particularly beneficial when the sorted columns show a high cardinality
+and are used as a filter for selective reads.
+
+The sort order is configured with the sorted by table property.
+Specify an array of one or more columns to use for sorting
+when creating the table. The following example configures the
+order_date column in the orders table in the customers schema in the example catalog::
+
+    CREATE TABLE example.customers.orders (
+        order_id BIGINT,
+        order_date DATE,
+        account_number BIGINT,
+        customer VARCHAR,
+        country VARCHAR)
+    WITH (sorted_by = ARRAY['order_date'])
+
+Sorting can be combined with partitioning on the same column. For example::
+
+    CREATE TABLE example.customers.orders (
+        order_id BIGINT,
+        order_date DATE,
+        account_number BIGINT,
+        customer VARCHAR,
+        country VARCHAR)
+    WITH (
+        partitioning = ARRAY['month(order_date)'],
+        sorted_by = ARRAY['order_date']
+    )
+
+You can disable sorted writing with the session property sorted_writing_enabled set to false.
+
+While the Iceberg specification supports a number of transforms on sort columns, they are
```

Comment:
I would remove that .. we try to document only what works .. not all the different exceptions and limitations..

Reply:
Also .. what do you mean .. how could you even have a transforms .. use a transforming function in the table property? Would that not just fail with a syntax error?

Reply:
You can define them in Spark

Reply:
I would still remove that..